helpers.js

```diff
@@ -14,48 +16,90 @@ const shouldUseYarn = () => {
   }
 };
 
-exports.installDependencies = packages => {
+const yarnPrompt = {
```

Comment:
I feel like there should be a check to see if the user has chosen this option prior, and if so it won't prompt their choice again the next time they run ```blix add```